### PR TITLE
dev-qt/qtwebengine: Fix detecting audio backends.

### PR DESCRIPTION
**Pull request by Chiitoo**
The upstream change doesn't really work for us, so we'll make
it work with us.

http://code.qt.io/cgit/qt/qtwebengine.git/commit/?h=5.7.1&id=cf7d82a79dff4ea5c53e0908ed5c13ce1f87796d

Gentoo-Bug: https://bugs.gentoo.org/603498

Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

**Comment by Chiitoo** [comment]
If nothing else, the single quotes might be unnecessary... but it's too late to change it now, because of things!

File: dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	qt_use_disable_mod widgets widgets src/src.pro
+
+	qt5-build_src_prepare
+}
+
+src_configure() {
+	export NINJA_PATH=/usr/bin/ninja
+
+	local myqmakeargs=(
+		'WEBENGINE_CONFIG+=use_alsa'
```
Comment (Pesa) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: well at this point we could add a USE flag for `alsa` as well? and yes the quotes are redundant

**Comment by Chiitoo** [comment]
Shows how sleepy I was getting...

Should be better now.

Also edited the patch comment parts a little, as well as its file name, and added a flag for ALSA.  I was thinking of adding it before as well, but wasn't too sure about it.

File: dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+		$(usex system-ffmpeg 'WEBENGINE_CONFIG+=use_system_ffmpeg' '')
+		$(usex system-icu 'WEBENGINE_CONFIG+=use_system_icu' '')
+	)
+	qt5-build_src_configure
+}
+
+src_install() {
+	qt5-build_src_install
+
+	pax-mark m "${D%/}${QT5_LIBEXECDIR}"/QtWebEngineProcess
```
Comment (SoapGentoo) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: will this not fail on prefix?

File: dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+		src/core/core_common.pri \
+		src/core/core_gyp_generator.pro
+
+	qt_use_disable_mod widgets widgets src/src.pro
+
+	qt5-build_src_prepare
+}
+
+src_configure() {
+	export NINJA_PATH=/usr/bin/ninja
```
Comment (SoapGentoo) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: Prefix fail?

Comment (Pesa) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: probably yeah, should be fixed in a separate commit though

File: dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild (hunk 2 above)
Comment (Pesa) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: why? `QT5_LIBEXECDIR` already contains `EPREFIX`

Comment (SoapGentoo) [inline dev-qt/qtwebengine/qtwebengine-5.7.1-r1.ebuild]: oh, missed that, alright then

**Comment by SoapGentoo** [comment]
@Pesa could you recheck?
